src/Namotion.Reflection/XmlDocsExtensions.cs

```diff
@@ -314,10 +314,10 @@ public static string ToXmlDocsContent(this XElement element)
                                 }
                             }
                         }
-                        else if (e.Name == "paramref")
-                        {
-                            var nameAttribute = e.Attribute("name");
-                            value.Append(nameAttribute?.Value ?? e.Value);
+                        else if (e.Name == "paramref")
+                        {
+                            var nameAttribute = e.Attribute("name");
+                            value.Append(nameAttribute?.Value ?? e.Value);
```

Comment:
I dont see a change here? Can you revert that if there is no change needed here?

Reply:
Reverted.